### PR TITLE
Add resources from the SCART website

### PR DESCRIPTION
**Pull request by kfrn**
I've added several articles and case studies from the Scart.be website. Just material in English, at this stage.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -60,8 +61,13 @@ Contributions are welcomed!
...
 * **ZKM | Center for Art and Media Karlsruhe** (2010-2012), [digital art conservation: 10 case studies](http://www.digitalartconservation.org/index.php/en/case-studies.html)
+* **Nederlands Instituut voor Mediakunst (NIMk)** (2011), [_I/Eye_ (1993/2011, Bill Spinhoven)](https://www.scart.be/?q=en/content/case-study-report-ieye-19932011-bill-spinhoven)
+* **Emanuel Lorrain (PACKED vzw)** (c.2013), [The preservation of Danny Matthys' video works](https://www.scart.be/?q=en/content/case-study-report%C2%A0-preservation-danny-matthys-video-works)
+* **Emanuel Lorrain (PACKED vzw)** (c.2013), [_Mondophrenetic™_ (2000, Herman Asselberghs, Els Opsomer, Rony Vissers)](https://www.scart.be/?q=en/content/case-study-report-mondophrenetic%E2%84%A2-2000-herman-asselberghs-els-opsomer-rony-vissers-0)
```
Comment (retokromer) [inline README.md:66]: Nit: some lines above we did use a space in `c. 2013`

Comment (kfrn) [reply README.md:66]: I've normalised the spacing in the next commit. I prefer it as `c.2013`, and there was only one place where there was a space inbetween the `c.` and the year.

Comment (kfrn) [reply README.md:66]: Thanks, @retokromer ! Eagle eyes as always

**Review by retokromer — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (retokromer) [reply README.md:66]: I personally prefer the other way, but as long as it is normalised I am fine with.

**Comment by retokromer** [comment]
Thanks, @kfrn, as usual!

**Pull request merged**